index.js

```diff
@@ -44,7 +46,15 @@ module.exports = {
 
   treeForAddonTestSupport: function() {
     var MergeTrees = require('broccoli-merge-trees');
-    return new MergeTrees(this._getDependencyTrees());
+    var tree = new MergeTrees(this._getDependencyTrees());
+
+    if (this._shouldPreprocessAddonTestSupport) {
+      return this.preprocessJs(tree, '/', this.name, {
```

Comment:
Need to replace `this.name` with `/`.  This comes from [here](https://github.com/ember-cli/ember-cli-preprocessor-registry/blob/master/preprocessors.js#L164) which has the following signature:

```js
module.exports.preprocessJs = function(/* tree, inputPath, outputPath, options */) {}
```

Basically, the way we have it currently, will move these modules to `ember-cli-qunit/{ember-qunit,ember-test-helpers}` instead of `{ember-qunit,ember-test-helpers}`.